src/main/java/mServer/crawler/Crawler.java

```diff
@@ -68,7 +70,7 @@ public void fertig(ListenerFilmeLadenEvent event) {
         }
         try {
             while (serverLaufen) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "true" [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2583)